<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FenClient</class>
 <widget class="QWidget" name="FenClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chat - Client</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>chat.png</normaloff>chat.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>IP сервера</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="serverIP">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>192.168.0.101</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Порт сервера:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="serverPort">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="minimum">
        <number>1024</number>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="value">
        <number>50885</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonConnection">
       <property name="text">
        <string>Подключиться</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="pixmap">
          <pixmap>chat.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Chat</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="messagesList">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Ник</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pseudo">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Сообщение</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="message"/>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSend">
       <property name="text">
        <string>Отправить</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>chat.png</normaloff>chat.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
